implementations/rust/ockam/ockam_command/src/enroll.rs

```diff
@@ -47,7 +47,7 @@ async fn rpc(ctx: Context, (opts, cmd): (CommandGlobalOpts, EnrollCommand)) -> R
 }
 
 async fn run_impl(ctx: &Context, opts: CommandGlobalOpts, cmd: EnrollCommand) -> Result<()> {
-    let node_name = start_embedded_node(ctx, &opts, None).await?;
+    let node_name = start_embedded_node(ctx, &opts, None, None).await?;
```

Comment:
Hopefully this function signature doesn't grow much wider - maybe an opportunity for builder pattern in a later refactor?